edx_sga/sga.py

```diff
@@ -135,12 +135,36 @@ def max_score(self):
         """
         return self.points
 
+    def _serialize_opaque_key(self, key):
+        """
+        Gracefully handle opaque keys, both before and after the transition.
+        https://github.com/edx/edx-platform/wiki/Opaque-Keys-(Locators)
+
+        Args:
+            key (unicode or OpaqueKey subclass): The key to serialize.
+
+        Returns:
+            unicode
+
+        """
+        if hasattr(key, 'to_deprecated_string'):
```

Comment:
@amir-qayyum-khan You're correct! I had copy-pasta'd this code from elsewhere - but it'd only be necessary in a pre-opaque-keys time. I'll change it.